modules/openshift-developer-cli-connecting-the-database.adoc

```diff
@@ -29,13 +29,13 @@ mongodb-1-gsznc                     1/1       Running   0          28m
 nodejs-nodejs-ex-mhbb-app-4-vkn9l   1/1       Running   0          1m
 
 $ oc rsh nodejs-nodejs-ex-mhbb-app-4-vkn9l
-sh-4.3$ env
+sh-4.4$ env
```

Comment:
And this file should be reverted

Reply:
See https://github.com/openshift/openshift-docs/pull/21456/files#r414132734